### PR TITLE
Default blueprint updates for availale_topo.yml

### PR DESCRIPTION
**Pull request by naraiza**

### CODE REVIEW

File: topologies/available_topo.yml [hunk 1]
```diff
@@ -1,32 +1,37 @@
+# topology_default - key used to set the default topology to use
```
Comment (devrobo) [inline topologies/available_topo.yml]: Is this the correct key name?

File: topologies/available_topo.yml [hunk 2]
```diff
@@ -1,32 +1,37 @@
...
+#  - If tag has no valid value or is missing, the first entry in topologies will be used as the default
+#
+# topologies - array of topologies to use
+# - Topology_Name: Must be less than 50 characters. Anything larger will be truncated
+# - Topology_File: Must include path to yml file in topologies directory
```
Comment (devrobo) [inline topologies/available_topo.yml]: If the path is invalid the entire topology entry is ignored.

File: topologies/available_topo.yml (hunk 1 above)
Comment (naraiza) [reply topologies/available_topo.yml]: Corrected

File: topologies/available_topo.yml (hunk 2 above)
Comment (naraiza) [reply topologies/available_topo.yml]: Corrected

**Pull request merged**